tests/automation/disappearing_messages.spec.ts

```diff
@@ -186,11 +200,15 @@ test_group_Alice_2W_Bob_1W_Charlie_1W(
 test_Alice_2W(
   'Disappear after send note to self',
   async ({ alice, aliceWindow1, aliceWindow2 }) => {
+    const disappearingMessagesType = 'disappear-after-send-option';
+    const timeOption: DMTimeOption = 'time-option-10-seconds';
+    const disappearAction: DisappearActions = 'sent';
     const testMessage = 'Message to test note to self';
     const testMessageDisappear = 'Message testing disappearing messages';
+    const formattedTime = await formatTimeOption(timeOption);
```

Comment:
`formatTimeOption` doesn't need to be async, so you'll have to remove those await once you made `formatTimeOption` not async